### PR TITLE
Lesson 4.1 technical review

### PR DESCRIPTION
**Pull request by andrewk**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -26,51 +26,49 @@ textChanged = () => {
 ## Arrow functions
 
 You may notice some funny looking syntax there. What's with this fat arrow
-business `=>`? That's some ES6 syntactical sugar that automatically binds
```
Comment (andrewk) [inline readme.md:29]: Arrow functions aren't syntactical sugar.

File: readme.md [hunk 2]
```diff
@@ -26,51 +26,49 @@ textChanged = () => {
...
-So now that we have a reference to the component, we can tell it to update:
+Now that we have a reference to the component, we can tell it to update:
 ```es6
 textChanged = () => {
   this.refs.readingTime.updateReadingTime()
 }
 ```
 
-When you add a ref to a component, that component is then added to a `refs`
-array on the parent component. So we can access it through `this.refs` and
```
Comment (andrewk) [inline readme.md:56]: Array with named indexes? PHP? ;)

File: readme.md [hunk 3]
```diff
@@ -3,8 +3,8 @@
 So far we've built our little app to count the number of words in an existing
 article and display that in our widget. This works, but isn't very exciting
 and doesn't respond to any changes on the page. Let's use some event handlers
-to trigger updates on our `ReadingTime` widget. Open up the `ReactReadingTime`
-component and add an `onChange` handler to the textarea.
+to trigger updates on our `ReadingTime` widget. Open up the `ReadingTime`
```
Comment (dphaener) [inline readme.md:6]: This actually should be the `ReactReadingTime` component.

File: readme.md (hunk 2 above)
Comment (dphaener) [inline readme.md:56]: Heh, whoops. Not sure where I got that from. :)

**Pull request merged**